spk/mono/Makefile

```diff
@@ -1,15 +1,14 @@
 SPK_NAME = mono
-SPK_VERS = 5.20.1.34
-SPK_REV = 17
+SPK_VERS = 6.12.0.122
+SPK_REV = 18
 SPK_ICON = src/mono.png
 
 DEPENDS = cross/$(SPK_NAME)
 
-MAINTAINER = hg59
+MAINTAINER = digitalbox98
 DESCRIPTION = Cross platform, open source .NET development framework
 DISPLAY_NAME = Mono
 STARTABLE = no
-CHANGELOG = "1. Update to version 5.20.1.34<br/>2. Fix workaround for ARMv5 (88f628x archs)"
```

Comment:
please update the CHANGELOG and to not remove it

Reply:
The CHANGELOG now indicates the update for the new version